.github/workflows/release.yml

```diff
@@ -54,10 +54,23 @@ jobs:
       - name: Fetch Git tags
         run: git fetch --force --tags
 
-      - name: Setup Golang with Cache
```

Comment:
And here we have the inverse - does `magnetikonline/action-golang-cache` not work, is buggy, or...?

Reply:
See above comment